.travis.yml

```diff
@@ -135,13 +136,10 @@ jobs:
         - master
   - stage: smoke
     script: npm run test:integration:remote
+  - stage: update translations
```

Comment:
After re-reading the [conditional builds, jobs, and stages doc](https://docs.travis-ci.com/user/conditional-builds-stages-jobs/) I think we're mixing up jobs and stages here.
If we keep update translations as a stage, then to be consistent, this part should only define the script, and the condition should be in the stages section below.

Reply:
OK, take a look now